interview-questions.md

```diff
@@ -8,42 +8,43 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is object-oriented programming? How is it different than functional programming?
 
-Your answer:
+Your answer: Object oriented programming is programming which is centered on objects. I can't recall what functional programming is, so I would indicate how I would go through my resources to brush up on my understanding in an interview, which would be internet searches, the LEARN syllabus & leaning on my cohort. 
 
-Researched answer:
+Researched answer: Object oriented programming is programming that uses classes of objects and functions (or methods) that work on them. The classes are the blueprint used to create specific objects. Functional programming is different in that Object oriented programming stores data in objects, which is not possible with functional programming. Functional programming takes data and transforms it by making functions. Essentially, the methods for storing and manipulating the data are different.
 
 2. What is the difference between a Float and an Integer in Ruby?
 
-Your answer:
+Your answer: A float is a number that contains a decimal point, and an integer is a number without a decimal point.
 
-Researched answer:
+Researched answer: A float represents a 'real' number, a number that contains a fraction, represented with a decimal point. an integer is a whole number, represented with no decimal point.
 
 3. Ruby has an implicit return. What does that mean?
 
-Your answer:
+Your answer: This means that you don't always need to directly call for a return, it is implied within the language. 
 
-Researched answer:
+Researched answer: An implicit return in Ruby refers to the fact that every method has an assumed return method. The last evaluated statement in a method will return on it's own. 
 
 4. What is a block in Ruby?
 
-Your answer:
+Your answer: I can't recall what a block is in Ruby (other than a blocker, but I don't think thats what it's referring to!), so  I would indicate how I would go through my resources to brush up on my understanding in an interview, which would be internet searches, the LEARN syllabus & leaning on my cohort.
+
+Researched answer: This is an anonymous function, inside of a do-end statement or inline with curly braces. It can be passed into methods, and can also contain arguments (|denoted by two pipes on either side|). 
 
-Researched answer:
 
 5. What is an instance variable in Ruby?
 
-Your answer:
+Your answer: I can't recall what an instance variable is in Ruby, so  I would indicate how I would go through my resources to brush up on my understanding in an interview, which would be internet searches, the LEARN syllabus & leaning on my cohort.
 
-Researched answer:
+Researched answer: An instance variable is used to create variables that store specific data. These are defined inside of classes and are denoted by the usage of the @ symbol. They are visible anywhere within the scope of the object. 
```

Comment:
Extra content that you can share is where they are located in the class, how to assign values to it, how to reference them in custom methods within the class, etc